tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,247 @@
+			},
+		}
+
+	})
+
+	AfterAll(func() {
+		// Delete the dev and managed namespaces with all the resources created in them
+		Expect(framework.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+		Expect(framework.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+	})
+
+	var _ = Describe("creation of the 'Happy path for defult pipeline bundle resources without deployment", func() {
+
+		It("creates config map in the dev namespace.", func() {
+			_, err := framework.CommonController.CreateConfigMap(cm, devNamespace)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("create release plan in the dev namespace.", func() {
+			_, err := framework.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, managedNamespace, "")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates release strategy in the managed namespace.", func() {
+			_, err := framework.ReleaseController.CreateReleaseStrategy(releaseStrategyDefaultName, managedNamespace, releasePipelineNameDefault, releasePipelineBundleDefault, releaseStrategyPolicyDefault, releaseStrategyServiceAccountDefault, paramsReleaseStrategyDefault)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates release plan admission in the managed namespace.", func() {
+			_, err := framework.ReleaseController.CreateReleasePlanAdmission(targetReleasePlanAdmissionName, devNamespace, applicationNameDefault, managedNamespace, "", "", releaseStrategyDefaultName) //releaseEnvironment
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates EnterpriseContractPolicy in the managed namespace.", func(ctx SpecContext) {
+			_, err := framework.TektonController.CreateEnterpriseContractPolicy(releaseStrategyPolicyDefault, managedNamespace, defaultEcPolicy)
+			Expect(err).NotTo(HaveOccurred())
+		}, SpecTimeout(EnterpriseContractPolicyTimeout))
+
+		It("creates pvc in the managed namespace.", func() {
+			_, err := framework.TektonController.CreatePVCInAccessMode(releasePvcName, managedNamespace, corev1.ReadWriteOnce)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates sevice account in the manged namespace.", func() {
+			_, err := framework.CommonController.CreateServiceAccount(releaseStrategyServiceAccountDefault, managedNamespace, managednamespaceSecret)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates role in the managed namespace.", func() {
+			_, err := framework.CommonController.CreateRole(roleName, managedNamespace, roleRules)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates Role binding with a given name in managednamespace.", func() {
+			_, err := framework.CommonController.CreateRoleBinding("role-relase-service-account-binding", managedNamespace, "ServiceAccount", roleName, roleRefKind, "role-m6-service-account", "rbac.authorization.k8s.io")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates application with the given name in dev namespace.", func() {
```

Comment:
I have the same comment here as below. I would expect this to be in the BeforeAll since really setup requirements not really a specific test for release suite.

Reply:
Thanks Danny, 
see above